downloads/swan-lake-release-notes/swan-lake-alpha3/RELEASE_NOTE.md

```diff
@@ -2,41 +2,928 @@
+
+### Overview of Ballerina Swan Lake Alpha3
+
+The Ballerina Swan Lake Alpha3 release includes the language features planned for the Ballerina Swan Lake release. Moreover, this release includes improvements and bug fixes to the compiler, runtime, standard library, and developer tooling. This release note lists only the features and updates added after the Alpha2 release of Ballerina Swan Lake.
+
+- [Updating Ballerina](#updating-ballerina)
+    - [For Existing Users](#for-existing-users)
+    - [For New Users](#for-new-users)
+- [Highlights](#highlights)
+- [What is new in Ballerina Swan Lake Alpha3](#what-is-new-in-ballerina-swan-lake-alpha3)
+    - [Language](#language)
+    - [Runtime](#runtime)
+    - [Standard Library](#standard-library)
+    - [Code to Cloud](#code-to-cloud)
+    - [Developer Tools](#developer-tools)
+    - [Breaking Changes](#breaking-changes)
 
 ### Updating Ballerina
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+You can use the [Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to update to Ballerina Swan Lake Alpha3 as follows.
 
-#### For existing users
+#### For Existing Users
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/learn/tooling-guide/cli-tools/update-tool/). To do this, first, execute the command below to get the update tool updated to its latest version. 
 
-> $ ballerina dist update
+> `bal update`
 
-However, you need to use the following commands instead of the above if you have installed:
+If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Alpha3.
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+> `bal dist pull slalpha3`
 
-#### For new users
+#### For New Users
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
 ### Highlights
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+### What is new in Ballerina Swan Lake Alpha3
 
 #### Language
--
+
+##### Support for Module-level Variables with List, Mapping, and Error Binding Patterns
+
+Variable declarations with a list, mapping, and error binding patterns are now allowed at the module level. Unlike simple variables, these variables must be initialized in the declaration.
```

Comment:
Should be "with list, mapping, and error..." right? And is it "mapping, and error" or "mapping, or error"?

Reply:
Also, @MaryamZi is "simple variable" an ok term to use? 🤔

Reply:
Yeah, since we are talking about patterns (plural) we don't need the "a". I think we can use "or" too.

Re: simple variables, the other alternative is to use "capture variables" but I'm not sure if that is a term familiar to users. Since it is clear what "simple variables" is here with the context, shall we go ahead with it?

```suggestion
Variable declarations with list, mapping, or error binding patterns are now allowed at the module level. Unlike simple variables, these variables must be initialized in the declaration.
```

Or we can say

```suggestion
Variable declarations with list, mapping, and error binding patterns are now allowed at the module level. Unlike capture variables, these variables must be initialized in the declaration.
```

Reply:
Capture variables sound unfamiliar and weird. :D Let's stick to simple variable then.

Reply:
Done.